shop.py

```diff
@@ -0,0 +1,67 @@
+class NotEnoughMoneyError(Exception):
```

Comment:
💕Q3 - 12/12! You have all of the 8 components we were looking for successfully execute in your script, plus you have two exceptions (2/2) and your code conforms to function standards (2/2) - great script